netstandard/ref/System.Reflection.cs

```diff
@@ -430,6 +438,12 @@ public sealed partial class DefaultMemberAttribute : System.Attribute
         public DefaultMemberAttribute(string memberName) { }
         public string MemberName { get { throw null; } }
     }
+    public abstract partial class DispatchProxy
```

Comment:
This likely should be tied to the Reflection.Emit decision as you need some level of reflection emit to implement it.

Reply:
This is the same category as System.Linq.Expressions. The .NET Core implementation uses Reflection.Emit but you do not strictly need Reflection.Emit to implement it.

Reply:
Cool, so we're OK with approving it as part of reflection then?